### PR TITLE
Override toString method for rest dto classes

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

It is helpful if we want to log some info when using rest sdk.

### _How was this patch tested?_
Pass existing UT.

image

### CODE REVIEW

File: kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java [hunk 1]
```diff
@@ -170,4 +170,44 @@ public boolean equals(Object o) {
...
+        + kyuubiInstance
+        + '\''
+        + ", state='"
+        + state
+        + '\''
+        + ", createTime="
+        + createTime
+        + ", endTime="
+        + endTime
+        + '}';
```
Comment (ulysses-you) [inline kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java:211]: is it generated by idea ? why looks so complex..

Comment (turboFei) [reply kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java:211]: yes, all the toString methods are generated by IDEA.

And then execute `dev/reformat` to format the code style.

Comment (ulysses-you) [reply kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java:211]: magic style..

**Review by ulysses-you — APPROVED** [review]

File: kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java (hunk 1 above)
Comment (turboFei) [reply kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java:211]: @pan3793  suggested to use ReflectionToStringBuilder, let me have a try.

Comment (turboFei) [reply kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java]: image

**Review by pan3793 — APPROVED** [review]

**Comment by turboFei** [comment]
thanks @ulysses-you @pan3793 , merged to master

**Pull request closed without merging**